tests/test_freshdesk_start_date.py

```diff
@@ -1,181 +1,157 @@
 import os
-
-from tap_tester import connections, runner
+from tap_tester import connections, runner, LOGGER
 
 from base import FreshdeskBaseTest
 
 
 class FreshdeskStartDateTest(FreshdeskBaseTest):
+    """Test that the start_date configuration is respected"""
 
     start_date_1 = ""
     start_date_2 = ""
-    test_streams = {}
 
     @staticmethod
     def name():
         return "tap_tester_freshdesk_start_date_test"
 
-    def get_properties(self, original: bool = True):
-        """Configuration properties required for the tap."""
-        return_value = {
-            'start_date' : '2019-01-06T00:00:00Z',
-        }
-        if original:
-            return return_value
-
-        return_value["start_date"] = self.start_date
-        return return_value
-
     def test_run(self):
-        """Instantiate start date according to the desired data set and run the test"""
+        """   
+        • Verify that a sync with a later start date has at least one record synced
+          and less records than the 1st sync with a previous start date
+        • Verify that each stream has less records than the earlier start date sync
+        • Verify all data from later start data has bookmark values >= start_date
+        • Verify that the minimum bookmark sent to the target for the later start_date sync
+          is greater than or equal to the start date
+        • Verify by primary key values, that all records in the 1st sync are included in the 2nd sync.
+        """
 
         self.start_date_1 = self.get_properties().get('start_date')
-        self.start_date_2 = self.timedelta_formatted(self.start_date_1, days=3*365+34)
+        self.start_date_2 = "2022-07-19T00:00:00Z"
 
         self.start_date = self.start_date_1
 
-        # Excluding broken streams: time_settings, satisfaction_ratings
-        # TODO spike on the two 403 streams above
-        test_streams = {'agents', 'companies', 'groups', 'tickets', 'conversations', 'roles'}
-        self.test_streams = test_streams
-        obey_start_date_streams = {'agents', 'companies', 'groups', 'roles', 'tickets', 'conversations'}
+        start_date_1_epoch = self.dt_to_ts(self.start_date_1, self.START_DATE_FORMAT)
+        start_date_2_epoch = self.dt_to_ts(self.start_date_2, self.START_DATE_FORMAT)
+
+        # To collect "time_entries", "satisfaction_ratings" pro account is needed. Skipping them for now.
+        expected_streams = self.expected_streams() - {'satisfaction_ratings', 'time_entries'}
 
         ##########################################################################
         ### First Sync
         ##########################################################################
 
-        # instantiate connection
-        conn_id_1 = connections.ensure_connection(self)
+        # Instantiate connection
+        conn_id_1 = connections.ensure_connection(self, original_properties=False)
```

Comment:
We are using the original start_date here, so can we remove the `original_properties=False` from the `ensure_connection`

Reply:
Removed